overrides/python/torch/default.nix

```diff
@@ -9,12 +9,14 @@
   # use the autoAddOpenGLRunpathHook to add /run/opengl-driver/lib to the RPATH
   #   of all ELF files
   deps = {nixpkgs, ...}: {
-    inherit (nixpkgs.cudaPackages) autoAddOpenGLRunpathHook;
+    inherit (nixpkgs.stdenv) isLinux;
+    autoAddOpenGLRunpathHook = lib.optionalAttribute nixpkgs.stdenv.isLinux nixpkgs.cudaPackages.autoAddOpenGLRunpathHook;
```

Comment:
I don't think `lib.optionalAttribute` is a thing?

Reply:
I don't know how I messed that up! I will look into and fix.